Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboarding/VehicleRegistrationCertificate.hs

```diff
@@ -468,7 +474,8 @@ validateRCStatus mbVariant rcconfigs rcInsurenceConfigs expiry insuranceValidity
           let validCOVsCheck = rcconfigs.vehicleClassCheckType
           let (isCOVValid, variant) = maybe (False, Nothing) (isValidCOVRC vehicleCategory capacity manufacturer vehicleClassVariantMap validCOVsCheck) (cov <|> vehicleCategory)
           let validInsurance = (not rcInsurenceConfigs.checkExpiry) || maybe False (now <) insuranceValidity
-          if ((not rcconfigs.checkExpiry) || now < expiry) && isCOVValid && validInsurance then (Domain.VALID, variant) else (Domain.INVALID, variant)
+          let isValidVariantForGovtData = maybe False (elem variantClassificationList) bodyType && Just 4 == capacity
```

Comment:
Why this config is separated this onboarding configs. This case can also be handled using the same configs. Lets discuss it.